my_compassion/static/src/js/my2_new_letter_child_selector_image.js

```diff
@@ -74,6 +74,33 @@ function onChildSelect(event) {
     if (mountedComponent) {
         mountedComponent.setSelectedChildId(childId); // Call the method on the component instance
     }
+
+    const letterInput = document.getElementById("letter-input");
+    if (letterInput) {
+        letterInput.value = "";
+    }
+
+    const attachmentsInput = document.getElementById("letter-attachments");
+    if (attachmentsInput) {
+        attachmentsInput.value = "";
+    }
+
+    const uploadedFilesContainer = document.getElementById("uploaded-files-container");
+    if (uploadedFilesContainer) {
+        uploadedFilesContainer.innerHTML = "";
+    }
+
+    const selectedTemplate = document.getElementById("selected-template");
+    if (selectedTemplate) {
+        selectedTemplate.remove();
+    }
+
+    const childDropdown = document.getElementById("child-dropdown");
+    if (childDropdown) {
+        childDropdown.value = childId;
+    }
+
+    window.location.href = `/my2/children/letters/new?child_id=${childId}`;
```

Comment:
@jordyBSK I did rebase your code to the latest version of the MyCompassion2.0 branch. Is this something you added or a rebase mistake ?